src/sas/sasgui/perspectives/corfunc/media/corfunc_help.rst

```diff
@@ -187,8 +187,8 @@ The structural parameters extracted are:
 *   Long Period :math:`= L_p`
 *   Average Hard Block Thickness :math:`= L_c`
 *   Average Core Thickness :math:`= D_0`
-*   Average Interface Thickness :math:`\text{} = D_{tr}`
-*   Polydispersity :math:`= \Gamma_{\text{min}}/\Gamma_{\text{max}}`
+*   Average Interface Thickness :math:`\mathrm{} = D_{tr}`
```

Comment:
Why is this one not like the others?  I suggest removing the ```\mathrm{} ``` here.

Reply:
Good point